### ISSUE TITLE
Broken layout with Paragraphs

### ISSUE THREAD

**Issue by drupix** [backdrop-contrib/ckeditor5#125]
I all and happy new year!

With Paragraphs modules enabled layout is broken on edit content, the field that use ckeditor is about 2000-3000px width and a big part of it disappears to the right of the screen. I tried to find the problems playing with css but I can not find it.

Have someone else tried CK5 with Paragraphs? Did you have the same behaviors?

Thank for feedback.

**Comment by quicksketch** [comment]
Hi @drupix, could you post a screenshot? Like you say, it sounds like it might just be a CSS issue.

**Comment by quicksketch** [comment]
I tried to reproduce this with Paragraphs, in the embedded widget, but it seems to load just fine. This is with the out-of-box Seven theme:

![image](https://github.com/backdrop-contrib/ckeditor5/assets/100206/9f0a9e3a-de2e-4dba-95eb-64e9b2862fe1)

**Comment by laryn** [comment]
If this is with Gin, I saw that once but didn't get an issue filed at that moment and haven't reproduced it since. Either way, if you can help figure out how to replicate, I'm sure we can figure out what's going on.

![CleanShot 2024-01-10 at 18 20 52@2x](https://github.com/backdrop-contrib/ckeditor5/assets/871421/62fd5145-137e-4507-8665-90a8012c7092)

**Comment by drupix** [comment]
Hi, thanks for the quick reply!

I forgot to mention that I have nested Paragraphs, maybe that's the point !?

- my content type has a paragraph field with embedded widget that references paragraph "Section"
- the paragraph "Section" has 4 fields for his appearance (background image/position/color, spacer and Title) and a field "Content" that is another embedded widget that references other Paragraphs (actually only a Text field with CKEditor).

This works fine with CK4 on Basis or Gin themes but not with CK5. Here are 2 screenshots:

![ck4](https://github.com/backdrop-contrib/ckeditor5/assets/8409505/a8186baf-9c23-41f4-a576-e5582a5d9a92)

![ck5](https://github.com/backdrop-contrib/ckeditor5/assets/8409505/be0bd549-54be-401d-953e-20fb14499550)

**Comment by laryn** [comment]
@drupix It's something more than just the nested Paragraphs. I haven't duplicated your Paragraphs exactly but did test a nested setup and don't see it happening locally:
![image](https://github.com/backdrop-contrib/ckeditor5/assets/871421/42a44c8d-94bb-41ed-b1d2-2bce52677e8f)

**Comment by laryn** [comment]
Nevermind! As soon as I paste a long line of text into the nested CKEditor it explodes sideways... And it does happen in both Gin and Seven. Something to work from.

**Comment by laryn** [comment]
This line seems to be the culprit:

https://github.com/backdrop/backdrop/blob/d1218127af64248d9354cf7bf9a2054816d77e4b/core/modules/system/css/system.theme.css#L95

**Comment by laryn** [comment]
@drupix @quicksketch I've tested locally by just deleting that line 95 in `system.theme.css` and it fixes this issue. I don't see any immediate negative repercussions from removing that and I'm not 100% sure what it's original purpose was (it seems a little heavy-handed). 

```
tr.odd .form-item,
tr.even .form-item {
  margin-top: 0;
  margin-bottom: 0;
  // white-space: nowrap;
}
```

**@drupix Are you able to file this as an issue [in the core queue](https://github.com/backdrop/backdrop-issues/issues) (and file a PR for discussion/merge there)? It would be nice if possible to get a fix in before the release on Monday.**

**Comment by indigoxela** [comment]
Hi @drupix, many thanks for reporting, I was able to reproduce with only core involved and opened an issue in the core queue: https://github.com/backdrop/backdrop-issues/issues/6364

**Comment by drupix** [comment]
I'm a little bit late...

Thank you all for your quick reactions! I confirm that the fix in [backdrop/backdrop-issues/issues/6364](https://github.com/backdrop/backdrop-issues/issues/6364) works for me!

Many thanks.

PS: Heu... do I have to close this issue?

**Comment by laryn** [comment]
@drupix Great, glad the fix is working. Should be included in `1.27.0`. I think we can close this issue in the meantime. 👍
